### PR TITLE
OBI-2 crops.obic outdated 

### PR DESCRIPTION
**Pull request by BrentWHR**
Update crops.obic

### CODE REVIEW

**Comment by BrentWHR** [comment]
Still needs to be done:
* make `crops.obic` an alias of `crops_obic` **OR**
*  give up on save data with usethis and do it the old fashioned way so you can link object `crops.obic` to file "crops_obic.RData"

**Comment by BrentWHR** [comment]
> Still needs to be done:
> 
>     * make `crops.obic` an alias of `crops_obic` **OR**
> 
>     * give up on save data with usethis and do it the old fashioned way so you can link object `crops.obic` to file "crops_obic.RData"

I've given up. Using usethis to get an .rda file results in a file with the same name as the object (admittedly this makes sense). So we cannot have files with one name and objects with a slightly different name.

**Comment by BrentWHR** [comment]
Still needs to be done:
* N use norms (the nf_* columns) for `crops.obic[crop_code %in% c(6660, 6664)]`

File: data-raw/crops_obic/b_lu_brp.csv [hunk 1]
```diff
@@ -0,0 +1,461 @@
+B_LU_BRP,B_LU_NAME,B_REGION,B_LU,B_LU_WATERSTRESS_OBIC,B_LU_CROPROTATIONINTENSITY_OBIC,B_LU_EOM,B_LU_EOM_RESIDUE,B_LU_HC,B_LU_CULTCAT4,B_LU_CROPROTATION,B_LU_CRUMBLEABILITY_OBIC,B_LU_PCLASS_OBIC,B_LU_SEALINGCLASS,B_LU_NEVALCLASS_OBIC,B_LU_KEVALCLASS_OBIC,B_LU_MEASURE_OBIC,B_LU_BOTANICAL,B_LU_SEASON,B_LU_MAKKINK,B_LU_BBWP,B_LU_ARABLE_ER,B_LU_PRODUCTIVE_ER,B_LU_CULTIVATED_ER,B_LU_ROOTBALL_DIAMETER,B_LU_PLANT_DENSITY
```
Comment (SvenVw) [inline data-raw/crops_obic/b_lu_brp.csv:1]: Can the number of columns be reduced by removing not relevant columns, like _er?

File: vignettes/packages.bib [hunk 2]
```diff
@@ -28,7 +28,7 @@ @Manual{R-OBIC
   author = {Sven Verweij and Gerard Ros and Yuki Fujita and Brent Riechelman},
   year = {2022},
   note = {R package version 2.0.1},
-  url = {https://CRAN.R-project.org/package=OBIC},
+  url = {https://github.com/AgroCares/Open-Bodem-Index-Calculator},
```
Comment (SvenVw) [inline vignettes/packages.bib:31]: Please leave this to CRAN

**Review by SvenVw — CHANGES_REQUESTED** [review]

File: data-raw/crops_obic/b_lu_brp.csv (hunk 1 above)
Comment (BrentWHR) [reply data-raw/crops_obic/b_lu_brp.csv:1]: These are removed from the final product, `crops.obic` when preparing `crops.obic` in "data-rwa/crops_obic/crops_obic.R". It is my understanding that data in "data-raw" is not distributed.
 This csv file is sourced from `pandex::b_lu_brp`. In my opinion, the responsibility of trimming the eventual data should lay with the data using package, not the data providing package.

File: vignettes/packages.bib (hunk 2 above)
Comment (BrentWHR) [reply vignettes/packages.bib]: resolved in 967da56ec7cbbd9010713958cb5286ad8ec7d541

File: data-raw/crops_obic/b_lu_brp.csv (hunk 1 above)
Comment (SvenVw) [reply data-raw/crops_obic/b_lu_brp.csv:1]: Ok, then we can remove this file as it sourced from pandex?

Comment (BrentWHR) [reply data-raw/crops_obic/b_lu_brp.csv:1]: We will prepare a subset of b_lu_brp in pandex including the rows and columns required by OBIC to be readable in OBIC

**Review by SvenVw — APPROVED** [review]

**Pull request merged**